dtfs-central-api/.eslintrc.js

```diff
@@ -32,12 +33,21 @@ module.exports = {
       {
         functions: false
       }
+    ],
+    'import/extensions': [
+      'error',
+      'ignorePackages',
+      {
+        js: 'never',
+        ts: 'never'
+      }
```

Comment:
I added this to remove the errors from lines like `export * from './mock-utilisation-reports'`